build: use issue forms for issue creation (#21997)
angular/angular-cli: .github/PULL_REQUEST_TEMPLATE.md

## PR Checklist

> Please check if your PR fulfills the following requirements:

Reviewer: ```suggestion
Please check to confirm your PR fulfills the following requirements:
```

> <!-- Please check the one that applies using "x". -->

Reviewer: ```suggestion
<!-- Please check all that apply using "x". -->
```

> - [ ] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-commit-message-guidelines

Reviewer: ```suggestion
- [ ] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-commit-message-guidelines
```
